images/win/scripts/ImageHelpers/ChocoHelpers.ps1

```diff
@@ -29,4 +29,21 @@ function Choco-Install {
             }
         }
     }
+}
+
+function Get-ChocoPackageLatestVersion {

```

Comment:
nit:
```suggestion
function Get-LatestChocoPackageVersion {
```

Reply:
Sure

Reply:
Let's add another function just with body `Invoke-RestMethod "https://community.chocolatey.org/api/v2/Packages()?$filterQuery"` and param $url and filterQuery

Reply:
Do you mean left this one as it is and add one more like `"Send-RequestToCocolatey"` ?

Reply:
Updated both